src/Game/Game.js

```diff
@@ -72,9 +81,9 @@ export default function Game() {
   const moves = history.map((squares, move) => {
     let description;
     if (move > 0) {
-      description = 'Go to move #' + move;
+      description = "Go to move #" + move;
```

Comment:
There is a bug in play against computer mode when go to move is selected.

Reply:
<img width="242" alt="image" src="https://github.com/MuhammadAwaisGondal/TIcTacToe/assets/150733676/f7e86eb3-1716-4a63-83b0-484ee2014e3d">

Human is O and Computer is X, but with Go to move button, Human can play computer's move.